service/fx.go

```diff
@@ -50,6 +50,18 @@ type (
 		OperatorRPSRatio                   persistenceClient.OperatorRPSRatio
 		DynamicRateLimitingParams          persistenceClient.DynamicRateLimitingParams
 	}
+
+	GrpcServerOptionsParams struct {
+		fx.In
+
+		Logger                 log.Logger
+		RpcFactory             common.RPCFactory
+		RetryableInterceptor   *interceptor.RetryableInterceptor
+		TelemetryInterceptor   *interceptor.TelemetryInterceptor
+		RateLimitInterceptor   *interceptor.RateLimitInterceptor
+		TracingInterceptor     telemetry.ServerTraceInterceptor
+		AdditionalInterceptors []grpc.UnaryServerInterceptor `optional:"true"`
```

Comment:
Can you document the semantics of how these interceptors will be applied? I think it's worth noting that these are applied after everything else but before the retry/rate limit interceptors